addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animations.py

```diff
@@ -233,9 +233,9 @@ def __get_blender_actions(blender_object: bpy.types.Object,
                 for track in blender_object.data.shape_keys.animation_data.nla_tracks:
                     # Multi-strip tracks do not export correctly yet (they need to be baked),
                     # so skip them for now and only write single-strip tracks.
-                    if track.strips is None or len(track.strips) != 1:
+                    if track.strips is None or len([strip for strip in track.strips if strip.mute is False]) != 1:
                         continue
-                    for strip in track.strips:
+                    for strip in [strip for strip in track.strips if strip.mute is False]:
```

Comment:
Maybe we should store the non-muted strips in a new variable at the top of the loop, so we don't have to repeat `if strip.mute is False` twice here? It would be easy to forget on new references.